### PR TITLE
Add `enable` option to disable flushing

### PR DESCRIPTION
**Pull request by wlingke**
Addresses #116, follow up from #118

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -38,6 +39,12 @@ class Analytics {
     this.flushAt = Math.max(options.flushAt, 1) || 20
     this.flushInterval = options.flushInterval || 10000
     this.flushed = false
+    Object.defineProperty(this, 'enable', {
+      configurable: false,
+      writable: false,
+      enumerable: true,
+      value: options.enable === false ? options.enable : true
```
Comment (vadimdemedes) [inline index.js]: Could you replace that with `typeof options.enable === 'boolean'`?

**Comment by wlingke** [comment]
Isn't what's there and what you're suggesting the same outcome?

**Comment by vadimdemedes** [comment]
Yes, the outcome is the same, but that code reads differently and is quicker to understand its purpose.

**Comment by wlingke** [comment]
Okay updated

**Comment by vadimdemedes** [comment]
@wlingke Thanks!
@f2prateek Could you publish a new minor version? I don't have publish access to this package.

**Comment by f2prateek** [comment]
Released in `3.3.0-beta.2`!

**Pull request merged**